allensdk/brain_observatory/filtered_running_speed/README.md

```diff
@@ -0,0 +1,25 @@
+Extract running speed
+=====================
+Calculates an average running speed for the subject on each stimulus frame.
+
+
+Running
+-------
+```
+python -m allensdk.brain_observatory.extract_running_speed --input_json <path to input json> --output_json <path to output json>
+```
+See the schema file for detailed information about input json contents.
+
+
+Input data
+----------
+- Stimulus pickle : Written by camstim (http://aibspi/braintv/camstim). Contains information about the stimuli that were 
```

Comment:
Isn't the special thing about this one that it takes three different running speed input files?  If so, they all should be mentioned here

Reply:
I need to update the README.md file. Thanks.

Reply:
I've updated the the README.md file to reflect the three stimulus files.